runestone/activecode/js/activecode.js

```diff
@@ -1272,6 +1271,19 @@ Yet another is that there is an internal error.  The internal error message is:
         }
     }
 
+    async toggleAlert(){
```

Comment:
I see that you have declared this method as async, yet you are not using `await` for anything inside the function and you are not `await`-ing its use on line 269 in activecode.js

If there is reason to wait for the results from logBookEvent then use await there, if there is not reason to await anything then perhaps this function should not be declared as async ?